modules/data_release/ajax/FileUpload.php

```
@@ -15,11 +15,11 @@
 $DB   = \Database::singleton();
 $user = \User::singleton();
 
-if ($_POST['action'] == 'upload'
+if ($_GET['action'] == 'upload'
```

Same here, maybe fail early if `'action'` param is bad.